### PR TITLE
Add otu log

### PR DESCRIPTION
**Pull request by schangccg**
ticket: https://github.com/muccg/bpa-archive-ops/issues/515

### CODE REVIEW

File: bpaotu/bpaotu/models.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+import logging
+logger = logging.getLogger("rainbow")
+
+
+class ImportLog(models.Model):
```
Comment (grahame) [inline bpaotu/bpaotu/models.py]: can we rename this to ImportOntologyLog? Then if we have any other classes of bug we display, we can add them :-)

File: bpaotu/bpaotu/templates/bpaotu/otu_log.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<style>
```
Comment (grahame) [inline bpaotu/bpaotu/templates/bpaotu/otu_log.html]: could you inherit from the base template, or at least add a <body> and <title> and so on?

Comment (grahame) [reply bpaotu/bpaotu/templates/bpaotu/otu_log.html]: oh and add a heading

File: bpaotu/bpaotu/templates/bpaotu/otu_log.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        border-collapse: collapse;
+    }
+</style>
+
+<table>
+    <thead>
+        <td><b>Ontology Name</b></td>
+        <td><b>Mismatches</b></td>
+    </thead>
+
```
Comment (grahame) [inline bpaotu/bpaotu/templates/bpaotu/otu_log.html]: show the filename the error came from; sort this table by filename and then field name

File: bpaotu/bpaotu/models.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    ontology_name = models.CharField(max_length=300)
+    import_result = models.TextField(null=True, blank=True)
```
Comment (grahame) [inline bpaotu/bpaotu/models.py]: track the filename the error was in (and run `os.path.basename` on it)

Comment (schangccg) [reply bpaotu/bpaotu/models.py]: Fixed.

File: bpaotu/bpaotu/templates/bpaotu/otu_log.html (hunk 3 above)
Comment (schangccg) [reply bpaotu/bpaotu/templates/bpaotu/otu_log.html]: Fixed.

File: bpaotu/bpaotu/templates/bpaotu/otu_log.html (hunk 2 above)
Comment (schangccg) [reply bpaotu/bpaotu/templates/bpaotu/otu_log.html]: Fixed.

File: bpaotu/bpaotu/models.py (hunk 1 above)
Comment (schangccg) [reply bpaotu/bpaotu/models.py]: Fixed.

**Pull request merged**